### ISSUE TITLE
v3.2.3 --httpsnippet Erroring out - but no specific errors

### ISSUE THREAD

**Issue by bkuhl** [Mermade/widdershins#124]
Using --httpsnippet I'm not able to generate any code snippets at all.  

I'm already using `--verbose`.  How can I get more information on what the problem is?

```
$ npm install widdershins
$ node node_modules/widdershins/widdershins.js --verbose --httpsnippet --language_tabs=shell:Shell  --language_tabs=php:PHP  --language_tabs=java:Java  --language_tabs=javascript:Javascript --outfile=./docs/v1/widdershins/index.html.md ./docs/v1/main.yaml
+ widdershins@3.2.3
updated 1 package and moved 1 package in 10.551s
starting deref My API
finished deref
Compiling all doT templates...
Loaded def authentication.def
Loaded def authentication_none.def
Loaded def callbacks.def
Loaded def debug.def
Loaded def discovery.def
Loaded def footer.def
Loaded def links.def
Loaded def parameters.def
Loaded def responses.def
Loaded def security.def
Compiling code_go.dot to function
Compiling code_http.dot to function
Compiling code_java.dot to function
Compiling code_javascript.dot to function
Compiling code_nodejs.dot to function
Compiling code_python.dot to function
Compiling code_ruby.dot to function
Compiling code_shell.dot to function
Compiling main.dot to function
Compiling operation.dot to function
Compiling translations.dot to function
Error generating code sample using httpsnippet for shell 
Error generating code sample using httpsnippet for php 
Error generating code sample using httpsnippet for java 
Error generating code sample using httpsnippet for javascript
```

**Comment by MikeRalphson** [comment]
Are you able to share your OpenAPI definition in a gist?

**Comment by MikeRalphson** [comment]
Alternatively, please try `v3.2.4` (again, with the verbose flag). You should see the actual exception error message now.

**Comment by bkuhl** [comment]
No luck, looks like it added a space after it though.

```
ompiling translations.dot to function
Error generating code sample using httpsnippet for shell 

Error generating code sample using httpsnippet for php 

Error generating code sample using httpsnippet for java 

Error generating code sample using httpsnippet for javascript
```

Here's my file, with descriptions removed and things changed, since this is private for work:

```
openapi: 3.0.0
info:
  title: Waffle API
  description: >
    Just a long description for testing
  version: 1.0.0

servers:
  - url: https://mysite.com
    description: Production

tags:
  - name: "Waffles"

paths:
  /customers/{customerId}/extra-stuff:
    parameters:
      - name: customerId
        in: path
        required: true
        example: 1
        schema:
          type: integer
          format: int64
          minimum: 1
    get:
      summary: Returns extra stuff for a given customer.
      operationId: getCustomerExtraStuff
      tags:
        - "Waffles"
      parameters:
        - in: header
          name: Header 1
          description: 
          schema:
            type: integer
            format: int64
            minimum: 1
          required: true
          example: 1
        - in: header
          name: Header 2
          schema:
            type: string
          required: true
          example: hsapp
      responses:
        '200':
          description: '200 OK'
          content:
            application/json:
              schema:
                type: object
                properties:
                  customerId:
                    type: integer
                  companyId:
                    type: integer
                  apps:
                    type: array
                    items:
                      type: object
                      properties:
                        appId:
                          type: integer
                          example: 56971
                        companyAppId:
                          type: integer
                        results:
                          type: array
                          items:
                            type: object
                            properties:
                              customerEmail:
                                type: string
                              externalCustomerId:
                                type: string
                              data:
                                description: something
```

I guess it's a good thing I can't get $ref to render through widdershins or this would've been much harder to get.

**Comment by MikeRalphson** [comment]
Added a fix for non-string type header example values in httpsnippet mode. Just running CI tests.

**Comment by vfernandestoptal** [comment]
nice, you beat me to it :+1:

**Comment by MikeRalphson** [comment]
Fix should be in `v3.2.5` - would be great if you could confirm.

**Comment by bkuhl** [comment]
Bingo!  We're in business.  You all rock!
